pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -162,6 +181,9 @@ public void init(SegmentGeneratorConfig segmentCreationSpec, SegmentIndexCreatio
     }
 
     // Initialize creators for dictionary, forward index and inverted index
+    IndexingConfig indexingConfig = _config.getTableConfig().getIndexingConfig();
+    int rangeIndexVersion =
```

Comment:
(minor) IndexingConfig can never be null
```suggestion
    int rangeIndexVersion = _config.getTableConfig().getIndexingConfig().getRangeIndexVersion();
```

Reply:
Ack